config.py

```diff
@@ -1,7 +1,11 @@
 import os
+from distutils.util import strtobool
+from pathlib import Path
 
 
 class Config(object):
+    FLASK_ROOT = str(Path(__file__).parent)
+
     CONTACT_EMAIL = os.environ.get("CONTACT_EMAIL")
     CONTACT_PHONE = os.environ.get("CONTACT_PHONE")
     DEPARTMENT_NAME = os.environ.get("DEPARTMENT_NAME")
```

Comment:
Can we put default values on these? This will prevent us needed to set them in AWS.